idaes/core/control_volume0d.py

```diff
@@ -1531,112 +1531,243 @@ def _get_stream_table_contents(self, time_point=0):
+                     for p in phase_list])
+                iscale.set_scaling_factor(self.enthalpy_transfer, sf)
 
         if hasattr(self, "deltaP"):
-            for t, v in self.deltaP.items():
-                if iscale.get_scaling_factor(v) is None:
-                    s = iscale.get_scaling_factor(
-                        self.properties_in[t].pressure,
-                        default=1,
-                        warning=True)
-                    iscale.set_scaling_factor(v, 10*s)
+            if iscale.get_scaling_factor(self.deltaP) is None:
+                sf = 10 * iscale.get_scaling_factor(
+                    self.properties_in[0].pressure,
+                    default=1, warning=True)
+                iscale.set_scaling_factor(self.deltaP, sf)
 
-        # Material Holdup Constraints
+        # Transform constraints in order of appearance -----------------------
         if hasattr(self, "material_holdup_calculation"):
-            for i, c in self.material_holdup_calculation.items():
-                iscale.constraint_scaling_transform(
-                    c, iscale.get_scaling_factor(
-                        self.material_holdup[i], default=1, warning=True))
+            for (t, p, j), c in self.material_holdup_calculation.items():
+                sf = iscale.get_scaling_factor(self.material_holdup[t, p, j])
+                iscale.constraint_scaling_transform(c, sf)
 
-        if hasattr(self, "pressure_balance"):
-            for t, c in self.pressure_balance.items():
-                iscale.constraint_scaling_transform(
-                    c, iscale.get_scaling_factor(
-                        self.properties_in[t].pressure, default=1, warning=True))
+        if hasattr(self, "rate_reaction_stoichiometry_constraint"):
+            for (t, p, j), c in self.rate_reaction_stoichiometry_constraint.items():
+                sf = iscale.get_scaling_factor(
+                        self.rate_reaction_generation[t, p, j])
+                iscale.constraint_scaling_transform(c, sf)
+
+        if hasattr(self, "equilibrium_reaction_stoichiometry_constraint"):
+            for (t, p, j), c in self.equilibrium_reaction_stoichiometry_constraint.items():
+                sf = iscale.get_scaling_factor(
+                        self.equilibrium_reaction_generation[t, p, j])
+                iscale.constraint_scaling_transform(c, sf)
 
-        # Material Balance Constraints
         if hasattr(self, "material_balances"):
             mb_type = self._constructed_material_balance_type
             if mb_type == MaterialBalanceType.componentPhase:
                 for (t, p, j), c in self.material_balances.items():
                     sf = iscale.get_scaling_factor(
                         self.properties_in[t].get_material_flow_terms(p, j),
-                        default=1,
-                        warning=True)
+                        default=1, warning=True)
```

Comment:
Minor comment on PEP8 standards (as I understood them when I read it ages ago): if arguments are broken over multiple lines, then you should generally have one argument per line.